src/main/java/org/folio/dew/service/JobCommandsReceiverService.java

```diff
@@ -122,6 +123,8 @@ public void receiveStartJobCommand(JobCommand jobCommand, Acknowledgment acknowl
         addBulkEditJobCommand(jobCommand);
         if (BULK_EDIT_IDENTIFIERS.equals(jobCommand.getExportType()) || BULK_EDIT_UPDATE.equals(jobCommand.getExportType())) {
           acknowledgementRepository.addAcknowledgement(jobCommand.getId().toString(), acknowledgment);
+          FolioExecutionScopeExecutionContextManager.endFolioExecutionContext();
+          log.info("FOLIO context closed.");
```

Comment:
1) Can this be in DEBUG level? 2) Can this logging be inside endFolioExecutionContext() method?

Reply:
1) The info log level is used as the same level was used to begin folio context in the module. 2)  FolioExecutionScopeExecutionContextManager from folio-spring-base library does not log begin or end context so there are no extra logs